modules/olmv1-red-hat-catalogs.adoc

```diff
@@ -28,7 +28,9 @@ spec:
     image:
       ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
       pullSecret: <pull_secret_name>
+      pollInterval: 24h <1>
```

Comment:
same as previous suggestion for using placeholder unless this is for a specific scenario.